src/resolvers.ts

```diff
@@ -1,114 +1,76 @@
-import crypto from 'crypto'
-// import { getDegreeById, getDegrees } from './services/degreeService'
-import { getFacilityById, searchFacilities } from './services/facilityService'
-import { getHealthcareProfessionalById, searchHealthcareProfessionals } from './services/healthcareProfessionalService'
-// import { getPhysicalAddressById, getPhysicalAddresses } from './services/physicalAddressService'
-// import { getSpecialtyById, getSpecialties } from './services/specialtyService'
-// import { getSpokenLanguageByIso, getSpokenLanguages } from './services/spokenLanguageService'
+import { addFacility, getFacilityById, searchFacilities } from './services/facilityService'
+import { addHealthcareProfessional, 
+    getHealthcareProfessionalById,
+    searchHealthcareProfessionals } from './services/healthcareProfessionalService'
 import {
+    Contact,
     Degree,
     Facility,
     Insurance,
     HealthcareProfessional,
     Specialty,
     LocaleNameInput,
     HealthcareProfessionalInput,
-    SpokenLanguage
+    SpokenLanguage,
+    LocaleName,
+    DegreeInput,
```

Comment:
what's your thoughts on why we created separate input entities vs exposing the regular entities?

Reply:
@ermish I'm not sure what you mean. I thought we had to create the input entities? I just followed the [doc examples](https://graphql.org/graphql-js/mutations-and-input-types/). Are you asking this because they're unnecessary?

Reply:
it depends on your design, but there's 3 options:
1 -  have separate entities for your api, business logic, and database
2 -  share entities for your business logic and db
3 -  share entities for your api and business logic

I usually prefer #3 or #1. If we want them separate for input that's fine. We should do this is across our api then for any inputs to be clearly suffixed with `Input`

Reply:
@ermish You're pretty much speaking a foreign language to me. 😵 Is it okay to keep the `Input`? They've existed in the schema for some time now for each Type. Do you see anywhere that the suffix is missing?

Reply:
TLDR, any input coming from the user should have the suffix `Input`. I think you have them covered here so far